indico/web/client/styles/base/_defaults.scss

```diff
@@ -46,7 +46,7 @@ $default-box-shadow-color: lighten($gray, $color-variation);
 }
 
 %font-size-body {
-    font-size: 13px;
+    font-size: 13px !important;
```

Comment:
Is there no way to get rid of this?

Reply:
I tried adding `body > :not(.rb-app-container) {` in the general body css to avoid using `!important` here but that ends up forcing css like `background-color` above classes that want to re-write that. Any other solution suggestions are welcome. :/